snippets/csharp/VS_Snippets_ADO.NET/DP Custom CopyToDataTable Examples/CS/Program.cs

```diff
@@ -360,10 +357,7 @@ public DataTable Shred(IEnumerable<T> source, DataTable table, LoadOption? optio
         public DataTable ShredPrimitive(IEnumerable<T> source, DataTable table, LoadOption? options)
         {
             // Create a new table if the input table is null.
-            if (table == null)
-            {
-                table = new DataTable(typeof(T).Name);
-            }
+			table ??= new DataTable(typeof(T).Name);
```

Comment:
It looks like tabs vs. spaces on indentation here:

```suggestion
            table ??= new DataTable(typeof(T).Name);
```